libkbfs/kbfs_ops.go

```diff
@@ -320,7 +320,11 @@ func (fs *KBFSOpsStandard) getMDByHandle(ctx context.Context,
 		rmd = fbo.getHead(lState)
 	}
 	if rmd != (ImmutableRootMetadata{}) {
-		return rmd, nil
+		if !fbo.identifyDone && getExtendedIdentify(ctx).behavior.AlwaysRunIdentify() {
```

Comment:
I think the bug here is that on line 320 `getHead` is called. That should probably be replaced with `getMDForReadNeedIdentify`, which does a bunch of checks, one of which is the identify.

Can you also add a comment to `getHead` saying that it shouldn't be called outside of `folder_branch_ops.go`?